stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1029,37 +1029,77 @@ LogicalResult DotOp::verify() {
 // DotGeneralOp
 //===----------------------------------------------------------------------===//
 
-LogicalResult DotGeneralOp::verify() {
-  auto dimNumbers = this->dot_dimension_numbers();
+LogicalResult DotGeneralOp::reifyReturnTypeShapes(
```

Comment:
I see that originally we had `verify` which was followed by `reifyReturnTypeShapes`. Now we have `reifyReturnTypeShapes` which is followed by `inferReturnTypeComponents`.

Let's reorder the function, so that `reifyReturnTypeShapes` comes second again. This will allow the diff to be more meaningful.